.github/workflows/ci.yml

```diff
@@ -0,0 +1,96 @@
+name: ci-workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'Repository info'
+        required: true
+      branch:
+        description: 'Name of the branch'
+        required: true
+      target_repository:
+        description: 'Repository name of the terratest'
+        required: true
+      target_pr:
+        description: 'PR number on the terratest repo'
+        required: true
+
+jobs:
+  ci-job:
+    runs-on: [ubuntu-latest]
+    steps:
+      - name: checkout to repo
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.inputs.repo }}
+          ref: ${{ github.event.inputs.branch }}
```

Comment:
How does the code from the official Terratest repo end up in the fork?

Reply:
This should be normal process for fork development where once the fork is created, you would setup official repo as upstream and pull/merge changes from upstream master.  This will be a required step for PRs from forks to pull latest before submitting/approving PR.

Reply:
also, I added a comment block to clarify the fork->develop->pr flow

https://github.com/gruntwork-io/terratest/pull/594/files#diff-e9f950f17198d3d5e3122a44230a09b9R3-R16